lisp/forge-gitee.el

```diff
@@ -0,0 +1,38 @@
+;;; forge-bitbucket.el --- Bitbucket support      -*- lexical-binding: t -*-
```

Comment:
You forgot to adjust this. When you edit this make sure `-*- lexical-binding: t -*-` begins at the same column as before.